app/lib/actions/katello/content_view_puppet_environment/clear.rb

```diff
@@ -3,7 +3,7 @@ module Katello
     module ContentViewPuppetEnvironment
       class Clear < Actions::Base
         def plan(repo)
-          plan_action(Pulp::Repository::RemovePuppetModule, pulp_id: repo.pulp_id)
+          plan_action(Pulp::Repository::RemoveUnits, repo_id: repo.id)
```

Comment:
In this case, repo here likely isn't a repository, its a content_view_puppet_environment.  You probably want to do something like this:  https://github.com/Katello/katello/blob/master/app/lib/actions/pulp/repository/create.rb#L21-L25  in Pulp::Repository::RemoveUnits

Reply:
oh i see, its already handled, ignore me :)